_resources/port1.0/group/ocaml-1.1.tcl

```diff
@@ -579,7 +579,15 @@ destroot {
     }
 }
 
+# Most of our OCaml ports do not have test dependencies,
+# so disable tests by default.
+default test.run    no
+
 test {
+    if {![option test.run]} {
+        ui_info "Tests are disabled."
+        return
+    }
```

Comment:
I am not sure I understand why this is needed. Based on the change in macports-base and the comments from @jmroot for that commit, there should only be a default check to test whether files are installed for the architecture that was specified. That should work here as well and no changes are needed if I am understanding this correctly. Otherwise, that change in base would have "broken" every port that hasn't enabled tests are specified all required test dependencies - I sincerely doubt that that is the case...

Reply:
@reneeotten Thanks for raising this point, let me try dropping that and see if it works.

(Will do as soon as back to my machine.)

Reply:
P. S. Though if I understand correctly, it will be broken here otherwise, since OCaml PG defines its own test stage.

Reply:
oh, well that could be... that seems an issue with the PG then that should probably be fixed.

Reply:
I am afraid I cannot rewrite the PG here, OCaml stuff is complicated since it uses several different build systems.

@catap @pmetzger Guys, maybe you could chime in? Any ideas? Your know OCaml stuff much better.

Reply:
I don't see that you need to disable the tests. Just don't run them in the default build.

Reply:
Ok, I will check if it works now.